src/components/UI/common/NewsletterSignup.tsx

```diff
@@ -25,28 +69,43 @@ export const NewsletterSignup: FC = () => {
         </PageText>
       </Stack>
 
-      <Stack w='100%' mb={3}>
-        <Input
-          placeholder='Enter your email'
-          bg='white'
-          borderRadius={0}
-          borderColor='brand.border'
-          h='56px'
-        />
-      </Stack>
+      <Stack w='100%'>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <FormControl id='email' isRequired mb={3}>
+            <Input
+              type='email'
+              placeholder='Enter your email'
+              bg='white'
+              borderRadius={0}
+              borderColor='brand.border'
+              h='56px'
+              {...register('email', {
+                required: true,
+                pattern: {
+                  value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
+                  message: 'Invalid email address'
```

Comment:
When would this validation message display?

Reply:
It's an optional message in the case we want to display it below the input